src/file-1.xoo

```diff
@@ -0,0 +1,17 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:37159/organizations/org0/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:37159/project/issues?id=sample&pullRequest=6080&issues=AWekjFUsvebVPU54-Ca9&open=AWekjFUsvebVPU54-Ca9)

[ISSUE_KEY]: AWekjFUsvebVPU54-Ca9